Apps/frontend/src/views/Waves.svelte

```diff
@@ -102,6 +102,17 @@
     webGLPlot.update();
     window.requestAnimationFrame(newFrame);
   };
+
+  export const resetPlot = () => {
```

Comment:
Can we use this function also when we initialize the plot the first time? Removes code duplications at the top of the file.
We could e.g. just write `let xArr` and init it in here, same with xLast and channel_samples.

Reply:
Great idea!